### ISSUE TITLE
TypeScript 2.4 Enums

### ISSUE THREAD

**Issue by rgevrek** [vojtechhabarta/typescript-generator#166]
TypeScript 2.4 now allows enum members to contain string initializers.

```
enum Colors {
    Red = "RED",
    Green = "GREEN",
    Blue = "BLUE",
}
```

Now, those Java enums
`public enum { A, B, C }`
could be
`enum { A="A", B="B", C="C" }`
so that on boths sides - backend and frontend - the enums could be used against the web services.

**Comment by vojtechhabarta** [comment]
It is already implemented, see [Generate TypeScript 2.4 string enums from Java enums ](https://github.com/vojtechhabarta/typescript-generator/pull/144).

Maven sample:
``` xml
<configuration>
    ...
    <mapEnum>asEnum</mapEnum>
</configuration>
```

**Comment by vojtechhabarta** [comment]
Closing the issue, we can reopen it if you need more info.
